src/coreclr/vm/typestring.cpp

```diff
@@ -801,36 +801,33 @@ void TypeString::AppendType(TypeNameBuilder& tnb, TypeHandle ty, Instantiation t
             StackSString ss;
             AppendType(ss, retAndArgTypes[0], format);
 
-            SString ssOpening(SString::Literal, "(");
-            ss += ssOpening;
-        
+            ss.Append(SL("("));
```

Comment:
Can we just use the `Append` overloads that take `const WCHAR *` and `const WCHAR` instead of the `SString`?

Reply:
Good point. Let me change that.